github/Requester.py

```diff
@@ -349,6 +360,38 @@ def __init__(
         self.__userAgent = user_agent
         self.__verify = verify
 
+    def _must_refresh_token(self) -> bool:
```

Comment:
Should the token be refreshed some time before `expires_at` to make sure that after calling this method the request does not exceed `expired_at`? Or will e request with an expired token be retried anyway?

Reply:
I put a 20 seconds delay (see `ACCESS_TOKEN_REFRESH_THRESHOLD_SECONDS`) to make sure there is still enough time to make the request.

Reply:
right, I misread the logic.